specification/dsdl/grammar.tex

```diff
@@ -253,3 +253,12 @@ \subsection{Reserved identifiers}\label{sec:dsdl_reserved_identifiers}
     \texttt{lpt\textbackslash{}d}                      & \texttt{lpt9}      & Compatibility with Microsoft Windows \\
     \texttt{\_.*\_}                                    & \texttt{\_offset\_}& Special-purpose intrinsic entities \\
 \end{CyphalSimpleTable}
+
+\subsection{Reserved comment forms}\label{sec:dsdl_reserved_comment}
+
+Line comments that match the following regular expression are reserved for vendor-specific language extensions:
+\verb|^#\[.+\].*|
+
+\begin{remark}
+    For example, \verb|#[canadensis(enum_start)]|.
```

Comment:
Shouldn't this be minted? As-is, the period is ambiguous. Perhaps:

```
    \begin{minted}{python}
        #[canadensis(enum_start)] all of this line is part of the reserved form.
        # After the newline this comment is now a standard DSDL comment.
    \end{minted}
```

Reply:
Updated as you suggested.